torch/_dynamo/output_graph.py

```diff
@@ -1082,9 +1091,14 @@ def compile_and_call_fx_graph(self, tx, rv, root):
         ] = self.dynamo_flat_name_to_original_fqn.copy()
 
         graph_code_log.debug("%s", lazy_format_graph_code(name, gm))
+        torch._logging.trace_structured(
+            "dynamo_output_graph",
+            payload_fn=lambda: gm.print_readable(print_output=False),
+        )
         graph_tabular_log.debug("%s", lazy_format_graph_tabular(name, gm))
-        graph_sizes_log.debug(
-            "%s", LazyString(lambda: self.get_graph_sizes_log_str(name))
+        graph_sizes_log.debug("%s", LazyString(lambda: self.get_graph_sizes(name)))
+        torch._logging.trace_structured(
+            "dynamo_output_graph_sizes", lambda: self.get_graph_sizes_structured()
```

Comment:
I think this is where structured trace logs is needed most, as every time I debug internal models (dynamic shapes enabled), this is where I really want to have a parse or tools to help me check hypothesis!